oem.json

```diff
@@ -2870,8 +2870,10 @@
     },
     {
         "id": "com.miui.phrase",
-        "description": "Frequent Phrases\nNot sure how exactly it can be used but it is supposed to predict phrases you'll want to write.\nI don't know why it isn't handled in the keyboard app. This seems to be something else.\nIn any case it has access to internet, is linked to MiCloud and contains a weird CloudTelephonyManager java class in his code.\n",
-        "removal": "delete"
+        "label": "Frequent phrases",
+        "description": "MIUI context menu tool for pre-made text responses while chatting. It has access to the Internet, is linked to MiCloud and contains a weird CloudTelephonyManager class in its code.",
+        "removal": "caution",
+        "warning": "Disabling causes a crash when touching the \"Frequent phrases\" item."
```

Comment:
> Disabling causes a crash when touching the “Frequent phrases” item.

Where is it exactly located? In a keyboard? If so, you need to include the name of the keyboard.

Reply:
> Where is it exactly located? In a keyboard?

"description": "**MIUI context menu tool**...

Reply:
> Disabling causes a crash when touching the "Frequent phrases" item.

Which package crashes? Only their default keyboard or your _active_ keyboard including 3rd party installed keyboards?

Reply:
It's not for a keyboard. This is a custom MIUI system context menu. Frequent phrases appear with a long tap on the input fields, as in the screenshot above. 

The exception is thrown in the application that implements the text input field. Therefore, the application crashes, from the activity of which the context menu is called.

Reply:
MIUI should have checked the existence of a matching intent before producing the option in that menu. Anyway, thanks @GflowerG for explaining it fully. I think you should amend the warning to write something like this:

> Disabling causes a crash when tapping on the \"Frequent phrases\" item in the context menu.